cloud_native_security.md

```diff
@@ -0,0 +1,37 @@
+# CLOUD NATIVE SECURITY DAY
+One of the things we do as a community is a Cloud Native Security day, it is designed to bring together the cloud native security community together to discuss and share current challenges and solutions in cloud native security to get together in a vendor neutral space.
+
+Cloud Native Security Day (#cnsecurityday) is intended to drive collaboration, discussion, and knowledge sharing of cloud native security accomplishments and roadblocks. Get connected with others that are passionate about security. Learn from practitioners about pitfalls to avoid, hurdles to jump, and how to integrate security into your cloud native project, architecture, and enhance team awareness on security.
+
+# Here are some references to some of the past events & some of the issues that detail the organization/planning of these events:
+
+## 2016
+
+KubeCon + CloudNativeCon North America, Seattle, USA - November 8-9 2016
```

Comment:
nit: we can skip the Kubecon lines here probably

Reply:
I actually tried keeping this because, the event was a part of Kubecon, makes it look clean. But yeah removing also won't effect much actually